Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyComboBox.h

```diff
@@ -57,6 +57,26 @@ class Q_SLICER_MODULE_SUBJECTHIERARCHY_WIDGETS_EXPORT qMRMLSubjectHierarchyCombo
   /// Else, the popup tree appears below the combobox, like for a qMRMLNodeComboBox.
   Q_PROPERTY(bool alignPopupVertically READ alignPopupVertically WRITE setAlignPopupVertically)
 
+  /// Filter to show only items that contain any of the given attributes with this name. Empty by default
+  Q_PROPERTY(QStringList includeItemAttributeNamesFilter READ includeItemAttributeNamesFilter WRITE setIncludeItemAttributeNamesFilter)
+  /// Filter to show only items for data nodes that contain any of the given attributes with this name. Empty by default
+  Q_PROPERTY(QStringList includeNodeAttributeNamesFilter READ includeNodeAttributeNamesFilter WRITE setIncludeNodeAttributeNamesFilter)
+  /// Filter to hide items that contain any of the given attributes with this name. Empty by default
+  /// Overrides \sa includeItemAttributeNamesFilter
+  Q_PROPERTY(QStringList excludeItemAttributeNamesFilter READ excludeItemAttributeNamesFilter WRITE setExcludeItemAttributeNamesFilter)
+  /// Filter to hide items for data nodes that contain any of the given attributes with this name. Empty by default
+  /// Overrides \sa includeNodeAttributeNamesFilter
+  Q_PROPERTY(QStringList excludeNodeAttributeNamesFilter READ excludeNodeAttributeNamesFilter WRITE setExcludeNodeAttributeNamesFilter)
+
+  /// Filter to show only items that contain an attribute with this name. Empty by default
+  /// Note: Deprecated, kept only for backwards compatibility. Sets and returns the first attribute in \sa includeNodeAttributeNamesFilter
+  Q_PROPERTY(QString attributeNameFilter READ attributeNameFilter WRITE setAttributeNameFilter)
```

Comment:
I think converting something to a property changes how it can be accessed from Python (you cannot access it as class.member() and class.setMember() anymore, only as class.member). So, this change would break existing modules. It would be nice to make this change for consistency, but it could be better to leave it as is for backward compatibility.

Reply:
It has already been a property.